LICENCE.txt

```diff
@@ -0,0 +1,19 @@
+Licence
+-------
+
+This file is part of IATI Query Builder.
+
+IATI Query Builder is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+IATI Query Builder is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
```

Comment:
Would be good to include the full text of the license - see https://choosealicense.com/licenses/gpl-3.0/